raiden-cli/src/routes/testing.ts

```diff
@@ -1,12 +1,30 @@
-import { Router, Request, Response } from 'express';
+import { Router, Request, Response, NextFunction } from 'express';
+import { ErrorCodes } from 'raiden-ts';
+import { validateAddressParameter } from '../utils/validation';
 import { Cli } from '../types';
 
+async function mintTokens(this: Cli, request: Request, response: Response, next: NextFunction) {
+  try {
+    response.json({
+      transaction_hash: await this.raiden.mint(request.params.tokenAddress, request.body.value, {
+        to: request.body.to,
+      }),
+    });
```

Comment:
Nitpick: For me the idea of this endpoint is to mint the tokens (`POST` as active request) and then return the result. This is how my brain works: declarative. So I would first await the minting and then respond. Makes it easy to follow the code. Personally I would even "construct" the response object separately but I wouldn't push for it.

Reply:
Done.